rllib/algorithms/ppo/ppo.py

```diff
@@ -366,10 +368,29 @@ def training_step(self) -> ResultDict:
         # workers.
         if self.workers.num_remote_workers() > 0:
             with self._timers[SYNCH_WORKER_WEIGHTS_TIMER]:
-                self.workers.sync_weights(
-                    policies=policies_to_update,
-                    global_vars=global_vars,
-                )
+                if self.config.get("_enable_trainer_runner", False):
```

Comment:
object, not dicts

Reply:
you then, wouldn't need to write this branching again here?